app/src/main/java/params/com/statusView/StatusView.kt

```diff
@@ -296,6 +296,14 @@ class StatusView @JvmOverloads constructor(
      *  A boolean which decides if to draw labels or not
      */
 
+    var drawCount: Boolean by OnValidateProp(false){
+        setDrawingDimensions()
+    }
+
+    /**
+     *  A boolean which decides if labels should be written inside
+     */
+
     var drawLabels: Boolean by OnValidateProp(false){
```

Comment:
Actually I see you are reusing the old variable drawLabels for new functionality.
This might make sense, but on a newer version people already using the library would get unexpected views & might be confused.
Can you instead create a new variable  drawStatusInsideCircle